src/lib/eslint/ESLintEngine.ts

```diff
@@ -1,320 +1,24 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
The diff looks confusing here because of class renames. Much of the deleted part lives in BaseEslintEngine. This module contains the two concrete implementations for Javascript and Typescript - the only logic here is the instantiation of the correct strategy to use with the engine.